neurodamus/io/sonata_config.py

```diff
@@ -308,7 +308,7 @@ def parsedProjections(self):
             for edge_pop_name, edge_pop_config in edge_config["populations"].items():
                 edge_pop = self._circuit_conf.edge_population(edge_pop_name)
                 pop_type = edge_pop_config.get("type", "chemical")
-                # skip unhandled synapse type
```

Comment:
why did you remove the comment? outdated and missleading?

Reply:
IMO, it's clear from the code; the warning says the same thing.